src/ArgentPonyWarcraftClient/Models/GameDataApi/Achievement/AchievementMedia.cs

```diff
@@ -18,5 +18,11 @@ public class AchievementMedia
         /// </summary>
         [JsonPropertyName("assets")]
         public Asset[] Assets { get; set; }
+
+        /// <summary>
+        /// Gets the Id of the achievement that the media belongs to.
+        /// </summary>
+        [JsonPropertyName("id")]
+        public int Id { get; set; }
```

Comment:
Found as a result of touching up the `Asset.FileDataId` and figured I'd clean that while I was at it.